docker/setup.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
why are these commands in their own script instead of just RUN from the Dockerfile?